vercel/turbo
# Make link command use auth crate instead of config token

### Description
`turbo link` was using the `BaseCommand` config token which exclusively reads from `config.json`, instead of using the new auth crate to find the token. This PR fixes that and uses the auth crate to find the token.

### Testing Instructions
Ensure you don't have a `config.json` with a token in it in whatever `XDG_CONFIG_HOME` you have (where we keep our tokens), run a `turbo login` and/or `turbo login --sso-team <team>`, then run the `turbo link` command. It should properly link.

After that, to double make sure, go to the current version of the code (in `main`) and try the same. If you have the `auth.json` that we create, with a valid token, but no `config.json`, then `turbo link` should fail.

Closes TURBO-1976
